lib/internal/navigator.js

```diff
@@ -17,11 +18,18 @@ const {
   getAvailableParallelism,
 } = internalBinding('os');
 
+const {
+  getDefaultLocale,
+  getAvailableLocales,
+} = internalBinding('icu');
```

Comment:
We support building without ICU. Please make sure building without ICU does not fail